### PR TITLE
feat(track-view-exercise): try to guess track

### PR DESCRIPTION
**Pull request by seanfarley**
If using `exercism-modern-jump` before `exercism-modern-view-tracks`, then the current track won't be set. This change will attempt to guess the track by using the current buffer, if applicable.

### CODE REVIEW

File: exercism-modern.el [hunk 1]
```diff
@@ -229,10 +233,27 @@ Pass prefix BUFFER-PREFIX-ARG to prompt for a buffer instead."
 
 ;;;###autoload
 (defun exercism-modern-track-view-exercises ()
-  "Invoked from `exercism-modern-track-mode', load the exercises for a given track."
+  "Invoked from `exercism-modern-track-mode', load the exercises for a given track.
+
+Alternatively, can be invoked from a buffer visiting a exercism
+file in its workspace. The track is determined by the current
+buffer's relative location to the exercism configured workspace,
+e.g. a workspace configuation of \"~/foo/exercism\" will
```
Comment (elken) [inline exercism-modern.el:241]: Rather than escaping these just use single quotes

File: exercism-modern.el [hunk 2]
```diff
@@ -300,6 +321,7 @@ Pass prefix BUFFER-PREFIX-ARG to prompt for a buffer instead."
                                                        :file ,difficulty-svg))
                                                     (propertize blurb 'face text-face))))))
           tabulated-list-padding 4)
+    (message "") ;; clear any previous messages
```
Comment (elken) [inline exercism-modern.el:324]: Not sure this is needed

**Review by elken — CHANGES_REQUESTED** [review]

File: exercism-modern.el (hunk 2 above)
Comment (seanfarley) [reply exercism-modern.el:324]: Else the "downloading X track ..." will continue to stay in the minibuffer.

Comment (elken) [reply exercism-modern.el:324]: Oh, put that in the downloading method then

**Pull request closed without merging**